src/tests/ftest/launch.py

```diff
@@ -733,10 +763,26 @@ def main():
         args.slurm_install = True
         args.slurm_setup = True
         args.user_create = True
+        args.fuse_setup = True
         args.clear_mounts.append("/mnt/daos")
         args.clear_mounts.append("/mnt/daos0")
         args.clear_mounts.append("/mnt/daos1")
 
+    elif args.mode == "custom_a":
+        if args.agent_user is None:
+            # Run the agent with the current user by default
+            args.agent_user = getpass.getuser()
+        if os.environ.get("DAOS_TEST_LOG_DIR", args.test_log_dir) is None:
+            # Use a user-specific test log dir by default
+            args.test_log_dir = os.path.join(
+                os.sep, "var", "tmp", f"daos_testing_{args.agent_user}")
```

Comment:
I think it's better to always use `getpass.getuser()` here. If for whatever reason `agent_user` is different, I think the directory should be named for the running user
```suggestion
            args.test_log_dir = os.path.join(
                os.sep, "var", "tmp", f"daos_testing_{getpass.getuser()}")
```

Reply:
This has been addressed in https://github.com/daos-stack/daos/blob/46b0765eebb7f99463c0fcb8417e5ccea69344cc/src/tests/ftest/util/environment_utils.py#L164